ghre-rails/app/views/pages/_update_summary.html.erb

```diff
@@ -0,0 +1,36 @@
+<% shifted_date = today - 2.weeks %>
+
+<% if update_date >= shifted_date %>
+  <div class="update-summary">
+    <span class="govuk-heading-s update-summary-meta">updated <%= update_date.strftime('%-d %B') %></span>
+    <% if show_updates == "true" %>
+      <a href="?show_updates=false" class="govuk-link govuk-link--no-visited-state" data-qa="updates-hide">
```

Comment:
This can be replaced with this in the future (tests might need to be updated though, not 100%) `<%= link_to 'hide updates', url_for(show_updates: false), class: 'govuk-link govuk-link--no-visited-state', data: { qa: 'updates-hide' } %>`

i wouldn't worry about it for this PR, just more for learnings sake

Reply:
Got it cheers